op-chain-ops/Makefile

```diff
@@ -44,4 +44,8 @@ fuzz:
 	go test $(FUZZLDFLAGS) -run NOTAREALTEST -v -fuzztime 10s -fuzz=FuzzAliasing ./crossdomain
 	go test $(FUZZLDFLAGS) -run NOTAREALTEST -v -fuzztime 10s -fuzz=FuzzVersionedNonce ./crossdomain
 
-.PHONY: test fuzz op-deployer
\ No newline at end of file
+
+sync-standard-version:
+	curl -Lo ./deployer/opsm/standard-versions.toml https://raw.githubusercontent.com/ethereum-optimism/superchain-registry/refs/heads/main/validation/standard/standard-versions.toml
```

Comment:
When is this intended to be ran? Can we add some info to the PR description detailing how to use this new feature?

Reply:
We'll need to run it whenever the standard versions change.